### ISSUE TITLE
Organize votes in governance repo centrally

### ISSUE THREAD

**Issue by jezdez** [conda-incubator/governance#85]
There was some discussion in the steering council chat room about record-keeping and discoverability of onging and past votes, since the they are becoming harder to track down.

As such @jakirkham and me are suggesting to do:

- [ ] require votes to be submitted as PRs to a to-be-created `votes` directory in this repository
- [ ] automatically request review from the steering council GitHub team via CODEOWNERS
- [ ] automatically assign the https://github.com/conda-incubator/governance/labels/vote label
- [ ] update the governance policy to mention the use of pull requests (which may trigger a governance policy change vote)

We also may want to look into if there is already a service that would provide automated vote reporting for standard votes (read: public ones).

**Comment by jakirkham** [comment]
Thanks for writing this up Jannis! 🙏

Something else we might consider is including the due dates for votes. Perhaps this could be done with a project. Toyed with this a bit [here]( https://github.com/orgs/conda-incubator/projects/5/views/1 )

**Comment by jezdez** [comment]
Yeah, did you see see GitHub's new beta roadmaps? https://github.blog/changelog/2023-01-31-roadmap-in-projects-public-beta/

Playing with them here: https://github.com/orgs/conda/projects/15/views/4

**Comment by jakirkham** [comment]
Ooh that looks nice! Well timed too 😃

Yeah really like that idea 👍

**Comment by jezdez** [comment]
Following another snafu of GitHub hiding the votes of some of the steering council members in #136 as spam (due to all being "yes"), it would seem to be useful to find a different mechanism for voting.

@ocefpaf suggested not to use PRs (as they increase the barrier of effort needed) and instead use an issue template form that would be filled when a vote is conducted (pre-setting the vote label, etc) and that would have a list of all steering council members as a check-list, so they can select yes, no or abstain easily.

@ocefpaf Is that what you had in mind?
